modules/rest-api/openapi/openapi.yaml

```diff
@@ -386,6 +386,35 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /management/v1/deployment/units/consistentId/{consistentId}:
+    get:
+      tags:
+      - deployment
+      description: Status of units which deployed on node.
+      operationId: byConsistentId
+      parameters:
+      - name: consistentId
+        in: path
+        required: true
+        schema:
+          required:
+          - "true"
+          type: string
+      responses:
+        "200":
+          description: All statutes returned successful.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UnitStatus'
+        "500":
```

Comment:
I think in case of an incorrect consistent id (non-existing one, for example) we have to return something like Not Found or Bad Request.